recipe/meta.yaml

```diff
@@ -3,6 +3,9 @@
 {% set sha256 = "fe744bd9c92557f0a38027f19a419b6e6765a982f531d02e620c79eb1a97bae7" %}
 
```

Comment:
you need `{% set build = 1 %}` in here, right?

Reply:
indeed, thanks